src/providers/historical-telemetry-provider.js

```diff
@@ -100,8 +101,11 @@ export default class YamcsHistoricalTelemetryProvider {
         url += `&${sizeParam}=${size}`;
         url += `&order=${order}`;
 
-        return accumulateResults(url, responseKeyName, [], totalRequestSize)
-            .then(convertHistory);
+        return accumulateResults(url, { signal }, responseKeyName, [], totalRequestSize)
+            .then(convertHistory)
+            .catch((error) => {
+                console.log('error', error);
```

Comment:
I think we want to propagate any errors up. If aborting causes an error in `convertHistory`, then we should add some handling specifically for the abort case.  Suppressing errors at the telemetry provider level will prevent us from doing consistent handling at a higher level (which I suspect we're not doing right now, but anyway).